onboarding-enabler-java-sample-app/src/test/java/org/zowe/apiml/hwsjersey/health/HealthControllerTest.java

```diff
@@ -12,15 +12,15 @@
 
 import org.glassfish.jersey.server.ResourceConfig;
 import org.glassfish.jersey.test.JerseyTest;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.zowe.apiml.eurekaservice.model.Health;
 
 import javax.ws.rs.core.Application;
 import javax.ws.rs.core.Response;
 import java.util.HashMap;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 // WARN: JerseyTest does not support Junit 5
```

Comment:
This comment is probably outdated now that it switched to junit 5 classes?

Reply:
Thanks for finding